src/services/countries.js

```
@@ -0,0 +1,37 @@
+const axios = require('axios');
+
+const getCountries = async () => {
+  try {
+    const response = await axios.get(`${process.env.COUNTRY_BASE_URL}/countries`, {
+      headers: {
+        'X-CSCAPI-KEY': process.env.COUNTRY_API_KEY,
+      },
+    });
+    return response.data;
+  } catch (error) {
+    throw new Error('Error with getting countries');
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)